### PR TITLE
Candytars

### PR DESCRIPTION
**Pull request by HHogg**
screen shot 2017-02-13 at 17 35 41

### CODE REVIEW

File: src/components/avatar/Candytar.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  'chartreuse',
+  'amber',
+  'orange',
+  'brown',
+  'grey',
+];
+
+export default class Candytar extends Component {
+  static propTypes = {
+    color: PropTypes.oneOf([
```
Comment (thatguynamedandy) [inline src/components/avatar/Candytar.js]: The propTypes thing, hurts me every time.

File: src/components/avatar/Candytar.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    const classes = classnames('ax-candytar-placeholder', `ax-candytar-placeholder--${color}`);
+
+    /* eslint-disable max-len */
+    return (
+      <svg className={ classes } style={ style } viewBox="0 0 32 32">
+        <filter id="ax-candytar-placeholder__filter">
+          <feColorMatrix type="matrix" values=" 1 0 0 0 -.27 0 1 0 0 -.27 0 0 1 0 -.27 0 0 0 .55 0"/>
+        </filter>
+        <circle className="ax-candytar-placeholder--background" cx="16" cy="16" r="16"></circle>
+        <path className="ax-candytar-placeholder--body" d="M16,20 C12.1340068,20 9,16.8659932 9,13 C9,9.13400675 12.1340068,6 16,6 C19.8659932,6 23,9.13400675 23,13 C23,16.8659932 19.8659932,20 16,20 Z M6,28.4693659 L6,28.4693659 C7.72660626,24.654283 11.5666365,22 16.0266078,22 C20.486579,22 24.3266093,24.654283 26.0532155,28.4693659 C23.3099448,30.6780366 19.8226569,32 16.0266078,32 C12.2305587,32 8.74327068,30.6780366 6,28.4693659 Z"></path>
```
Comment (thatguynamedandy) [inline src/components/avatar/Candytar.js]: I've often wondered with SVG wether we need such extreme rounding, 7 decimal places. No big deal but less rounding might decrease file size ever so slightly and make the path more readable.

We should monitor things like the `feColorMatrix` and potentially put them into a common location if used elsewhere.

File: src/components/avatar/Candytar.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  render() {
+    const { picker, color = picker(colors), size } = this.props;
+    const style = { height: size, width: size };
+    const classes = classnames('ax-candytar-placeholder', `ax-candytar-placeholder--${color}`);
+
+    /* eslint-disable max-len */
+    return (
+      <svg className={ classes } style={ style } viewBox="0 0 32 32">
+        <filter id="ax-candytar-placeholder__filter">
+          <feColorMatrix type="matrix" values=" 1 0 0 0 -.27 0 1 0 0 -.27 0 0 1 0 -.27 0 0 0 .55 0"/>
```
Comment (pmsorhaindo) [inline src/components/avatar/Candytar.js]: SNABD, but it bugs me to see a matrix shown as a string of numbers with no delimiters. Also there is a preceding space..

File: src/components/avatar/Candytar.js (hunk 1 above)
Comment (HHogg) [inline src/components/avatar/Candytar.js]: Yeah the duplication kills me :( I'll create an issue to explore alternatives/fixes to docgens computed proptypes issue.

Comment (HHogg) [inline src/components/avatar/Candytar.js]: #79

File: src/components/avatar/Candytar.js (hunk 2 above)
Comment (HHogg) [inline src/components/avatar/Candytar.js]: I'm reluctant to play around with optimisations on SVGs at the moment just incase they end up changing how design intended them to look. But the file size benefits are definitely worth keeping an eye on! I've created an issue #81 to look into this.

Comment (HHogg) [inline src/components/avatar/Candytar.js]: `feColorMatrix` There will definitely be a place for this as a common component if design follow this similar darken pattern.

**Pull request merged**